constants/constants.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+// NotFoundExceptionCode constant represents a custom error code for a not found record represented by 1000
+const NotFoundExceptionCode = 1000
+
+// UnprocessableEntityExceptionCode constant represents a custom error code for an unprocessable entity error represented by 1001
+const UnprocessableEntityExceptionCode = 1001
+
+// ThirdPartAPIExceptionCode constant represents a custom error code for a third part API error represented by 1002
+const ThirdPartAPIExceptionCode = 1002
+
+// WritingCSVFileExceptionCode constant represents a custom error code for a CSV writting error represented by 1003
+const WritingCSVFileExceptionCode = 1003
```

Comment:
You could define all constants inside a const group.

```suggestion
const {
    // NotFoundExceptionCode constant represents a custom error code for a not found record represented by 1000
    NotFoundExceptionCode = 1000

    // UnprocessableEntityExceptionCode constant represents a custom error code for an unprocessable entity error represented by 1001
    UnprocessableEntityExceptionCode = 1001

    // ThirdPartAPIExceptionCode constant represents a custom error code for a third part API error represented by 1002
    ThirdPartAPIExceptionCode = 1002

    // WritingCSVFileExceptionCode constant represents a custom error code for a CSV writting error represented by 1003
    WritingCSVFileExceptionCode = 1003
}
```